poh/benches/poh.rs

```diff
@@ -111,15 +111,13 @@ fn bench_poh_recorder_record_transaction_index(bencher: &mut Bencher) {
         SanitizedTransaction::from_transaction_for_tests(test_tx()),
     ];
 
+    let txs: Vec<_> = txs.iter().map(|tx| tx.to_versioned_transaction()).collect();
```

Comment:
unrelated to other changes in PR - the previous timing was measuring the conversion `to_versioned_transaction`...which makes no sense to do here.